docs/faq.md

```diff
@@ -41,18 +42,20 @@ The Chrysalis release still depends on the Coordinator/Compass to run and issue
 
 ## When is a Transaction on the Network Considered Final/Irreversible?
 
-On average, confirmation times on the new network are around 10 seconds. Once a transaction is set to confirm it’s final transaction, you don’t have block confirmations like with blockchain.
+On average, confirmation times on the new network are around 10 seconds. Once a transaction is set to confirm its final transaction, you do not have block confirmations like with blockchain.
```

Comment:
Tbh I'd check this one with the devs, I don't understand what it is supposed to mean.

Reply:
👍